loader/loader.go

```diff
@@ -161,11 +161,15 @@ func (l *loader) GetMatchingReleasePlanAdmission(ctx context.Context, cli client
 // error will be returned.
 func (l *loader) GetMatchingReleasePlans(ctx context.Context, cli client.Client, releasePlanAdmission *v1alpha1.ReleasePlanAdmission) (*v1alpha1.ReleasePlanList, error) {
 	releasePlans := &v1alpha1.ReleasePlanList{}
-	err := cli.List(ctx, releasePlans,
-		client.InNamespace(releasePlanAdmission.Spec.Origin),
-		client.MatchingFields{"spec.target": releasePlanAdmission.Namespace})
-	if err != nil {
-		return nil, err
+	if releasePlanAdmission.Spec.Origin != "" {
```

Comment:
Same as above.

Reply:
It fits the rest of the code to follow David's suggestion. Mostly just commenting here to say I think this is the only function that needs to change in this PR (got lucky I guess)

Reply:
ACK, on it.